scripts/hid_configurator/README.rst

```diff
@@ -56,8 +56,13 @@ Complete the following steps:
 #. Select the appropriate HIDADPI library version for the used Windows system (either ``x86`` or ``x64``).
    Copy the :file:`hidapi.dll` file and paste it into either of the following directory:
 
-   * The directory where used Python executable is located.
-   * The :file:`Windows\\System32` directory, for example :file:`C:\\Windows\\System32`.
+   * The directory hid_configurator directory.
```

Comment:
The directory where `configurator_cli.py` script is located (to be more precise)?